mllib/src/main/scala/org/apache/spark/mllib/stat/test/ChiSqTest.scala

```diff
@@ -224,12 +170,11 @@ private[spark] object ChiSqTest extends Logging {
 
     if (results.size < numFeatures) {
```

Comment:
Seems the test suite doesn't cover this ```if (results.size < numFeatures)``` path? Is it worth adding the coverage?

Reply:
This is cover in `ChiSquareTestSuite` via case `test DataFrame of sparse points`.
The values of the last feature are always zero, it will trigger this `if (results.size < numFeatures)` path